coder_sniffer/Drupal/Test/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -506,3 +506,15 @@ class Small {
 function test37(array $matches, array $sub_key, $to) {
 
 }
+
+/**
+ * Yield from should be a recognised return statement.
+ *
+ * @return int
+ *   Integer value.
+ */
+function test38($a, $b) {
+  for ($i = 1; $i <= 3; $i++) {
+    yield from $i;
```

Comment:
This is confusing because $i is an integer. `yield from` should be used on arrays or iterators. Can we have an example that yields from an array instead?

Reply:
Good point, I've updated the test function to a proper yield from implementation.